heltour/tournament/models.py

```diff
@@ -1814,11 +1820,12 @@ def __init__(self, *args, **kwargs):
         self.initial_player_id = self.player_id
         self.initial_games_missed = self.games_missed
         
-    def _set_unavailable_for_season(self):
+    def _set_unavailable_for_season(self, skip_current=False):
         rounds = Round.objects.filter(season=self.season, is_completed=False)
         for r in rounds:
-            PlayerAvailability.objects.update_or_create(round=r, player=self.player,
-                                                    defaults={'is_available': False})
+            if not (skip_current and r.publish_pairings):
```

Comment:
I don't know why, but I find this if statement more confusing than if we inverted it and used `continue`

```python
    if skip_current and r.publish_pairings:
        continue
    PlayerAvailability.objects.update_or_create(round=r, player=self.player,
                                                 defaults={'is_available': False})


        
```

Reply:
done